gallery_dl/extractor/twitter.py

```diff
@@ -991,7 +995,6 @@ def __init__(self, extractor):
         }
 
         self._nsfw_warning = True
-        self._syndication = extractor.config("syndication")
```

Comment:
Change this line to
``` python
self._syndication = self.extractor.syndication
```
so you don't have to access `self.extractor` further down.